packages/js/product-editor/src/contexts/validation-context/README.md

```diff
@@ -0,0 +1,39 @@
+# Validations and error handling
+
+This directory contains some components and hooks used for validations in the product editor.
+
+See [Product Editor Development Handbook](../../../../../../docs/product-editor-development/README.md)
+for more information.
+
+## Add a link to error snackbar
+
+### What happens when there is an error in a field?
+
+1. Every field [will be validated](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/product-editor/src/contexts/validation-context/validation-provider.tsx#L87-L110).
+
+2. An object with the [fields with errors](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/product-editor/src/contexts/validation-context/validation-provider.tsx#L74) will be returned.
```

Comment:
```suggestion
2. An object consisting of the error/validation message, the context, and the validatorId will be returned ([link](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/product-editor/src/contexts/validation-context/validation-provider.tsx#L74) ).
The `context` contains the block Id, and the `validatorId` a unique ID for the validator specifically ( generally a prefix with the block id ).
```
What does this object look like, I see the **step 3** has an example. In this case I think step 3 should be a sub-bullet.